packages/cypress/cypress/utils/oc_commands/scripts/mlflow_log_run.py

```
@@ -0,0 +1,54 @@
+
+headers = {
+    "Content-Type": "application/json",
+    "X-MLFLOW-WORKSPACE": workspace,
+}
+
+
+def api_post(endpoint, body):
+    data = json.dumps(body).encode()
+    req = urllib.request.Request(
+        f"{mlflow_url}{endpoint}", data=data, headers=headers
+    )
```

Ruff (0.15.10)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)